tfhe/src/c_api/high_level_api/compact_list.rs

```diff
@@ -73,26 +73,6 @@ pub unsafe extern "C" fn compact_ciphertext_list_builder_build_packed(
     })
 }
 
-#[cfg(feature = "zk-pok-experimental")]
-#[no_mangle]
-pub unsafe extern "C" fn compact_ciphertext_list_builder_build_with_proof(
-    builder: *const CompactCiphertextListBuilder,
-    public_params: *const CompactPkePublicParams,
-    compute_load: ZkComputeLoad,
-    list: *mut *mut ProvenCompactCiphertextList,
-) -> c_int {
-    catch_panic(|| {
-        let builder = get_ref_checked(builder).unwrap();
-        let public_params = get_ref_checked(public_params).unwrap();
-        let inner = builder
-            .0
-            .build_with_proof(&public_params.0, compute_load.into())
-            .unwrap();
-
-        *list = Box::into_raw(Box::new(ProvenCompactCiphertextList(inner)));
-    })
-}
-
```

Comment:
Se we remove the non packed option for proven ciphertext list ?

In that case should `compact_ciphertext_list_builder_build_with_proof_packed` simply become `compact_ciphertext_list_builder_build_with_proof` ?

Reply:
it's the current design choice to keep the proofs always packed (asked JB), but as new ZKs will be developped, maybe people won't want to bootstrap to unpack afterwards, I'm fine with keeping the _packed in the name at the meoment